Note support status of Project and Smart Proxies (#280)
theforeman/foreman-documentation: guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc

endif::[]

.Multi-homed {Project} and {SmartProxy}

> Multi-homed {Project} is not supported.

Reviewer: I would keep this as a Satellite specific note. Support status is really a downstream only thing.

> Multi-homed {SmartProxy} is supported, to implement this, you can configure {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].

Reviewer: This too. In upstream you can run a multi-homed Smart Proxy (also without a load balancer) but downstream doesn't want to support it.

.Multi-homed {Project} and {SmartProxy}

ifeval::["{build}" == "satellite"]
Multi-homed {Project} is not supported.

Multi-homed {SmartProxy} is supported, to implement this, you can configure {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
endif::[]

ifeval::["{build}" != "satellite"]

> You can use multi-homed {SmartProxies} by configuring {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].

Reviewer: This implies it's the only way. Would something like this be more appropriate?
```suggestion
One way to use multi-homed {SmartProxies} is by configuring {SmartProxies} with a load balancer. For more information, see {BaseURL}configuring_capsules_with_a_load_balancer/index[Configuring {SmartProxies} with a Load Balancer].
```

Author: Sure this works perfect!
